### ISSUE TITLE
[RoC] [V25a] Accidental Dependency on IE and Chisel (Bedrock Drill Head & Bedrock Chisel class errors)

### ISSUE THREAD

**Issue by RealShadowFury** [ReikaKalseki/Reika_Mods_Issues#2378]
I just updated to 25a and the drill head class does not seem to exist. i tested this with just forge, DragonAPI and Rotarycraft installed,no other mods.

Here is the pastebin for the crash: https://pastebin.com/Y5Vnxq8c

**Comment by RealShadowFury** [comment]
specific mod versions:
Forge: 10.13.4.16.14
RotaryCraft: 1.7.10 V25a
DragonAPI: 1.7.10 V25a

**Comment by fruitistasty** [comment]
I can confirm, also have the exact issue.

**Comment by ninjanomnom** [comment]
Crafting with the delegate doesn't crash but it does fail to finish the recipe and spams the creation sound.

**Comment by RealShadowFury** [comment]
I did some investigating, it seems a dependency check isn't running.
the two bedrock things that break are:
the bedrock drill head for use with Imeersive engineering and,
the bedrock chisel for the chisel mod.

installing chisel and immersive engineering fixes these problems.

TL;DR: this version of Rotarycraft has a (most likely un-intentional) dependency on Immersive Engineering and the chisel mod.

**Comment by RealShadowFury** [comment]
@ninjanomnom,

> 
> 
> Crafting with the delegate doesn't crash but it does fail to finish the recipe and spams the creation sound.

 I think you've got the wrong issue, this is an RoC issue, not a Chromaticraft issue,
I think you are looking for issue #2379

**Comment by ReikaKalseki** [comment]
Fixed.
